packages/next/src/server/app-render/use-flight-response.tsx

```diff
@@ -3,6 +3,9 @@ import type { FlightResponseRef } from './flight-response-ref'
 import { readableStreamTee } from '../stream-utils/node-web-streams-helper'
 import { encodeText, decodeText } from '../stream-utils/encode-decode'
 import { htmlEscapeJsonString } from '../htmlescape'
+// @ts-ignore
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createFromReadableStream } from 'react-server-dom-webpack/client.edge'
```

Comment:
This was changed from a static import to dynamic require in https://github.com/vercel/next.js/pull/41510

You might want to check with @ijjk to see why this was changed (I'm guessing performance)